### PR TITLE
Minor: Make wrapper script non-zero exits fatal.

### PR DESCRIPTION
**Pull request by cevich**
How to test:

```bash
$ cat /tmp/test.yml 
---

- hosts: localhost
  gather_facts: False
  tasks:
      - debug:
      - fail:

$ ./venv-ansible-playbook.sh -i /dev/null -c local -l localhost /tmp/test.yml; echo $?

Using temporary $WORKSPACE="/tmp/tmp.Lc05NTxGjM" for execution environment.
Directory will be removed upon exit.  Export this variable with path
to an existing directory to preserve contents.

Bootstrapping trusted virtual environment, this may take a few minutes, depending on networking.
(logs:  "/tmp/tmp.Lc05NTxGjM/crio_venv_setup_log.txt")


Executing "/tmp/tmp.Lc05NTxGjM/.cri-o_venv/bin/ansible-playbook -i /dev/null -c local -l localhost /tmp/test.yml"

 [WARNING]: provided hosts list is empty, only localhost is available


PLAY [localhost] ****************************************************************************************

TASK [debug] ********************************************************************************************
ok: [localhost] => {
    "msg": "Hello world!"
}

TASK [fail] *********************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "failed": true, "msg": "Failed as requested from task"}
	to retry, use: --limit @/tmp/test.retry

PLAY RECAP **********************************************************************************************
localhost                  : ok=1    changed=0    unreachable=0    failed=1   

2
```

### CODE REVIEW

File: contrib/test/venv-ansible-playbook.sh [hunk 1]
```diff
@@ -10,6 +10,9 @@
 #               --verbose \
 #               $PWD/crio-integration-playbook.yaml
 
+# All errors are fatal
+set -e
```
Comment (mrunalp) [inline contrib/test/venv-ansible-playbook.sh]: :+1:

**Comment by TomSweeneyRedHat** [comment]
Assuming happy tests, LGTM

**Comment by cevich** [comment]
**Do not bracket-re-test-bracket** running job manually...

**Comment by cevich** [comment]
Manual run: https://hurl.corp.redhat.com/8c12067 (internal link)

**Comment by cevich** [comment]
@mrunalp I dunno where johnny is.  This change should also fix it, though I still have no idea why different hashes are coming in from when I run it manually 😖

**Comment by cevich** [comment]
Note: Has mismatch could be simply because pip is pulling down pre-compiled binaries for a different platform (somehow) - I was testing it under F25 but CI env. may be running something else.

In any case, merging this will move things forward quickly and not require updating the internal stuff.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**